search-app/src/cmr/search/site/data.clj

```diff
@@ -35,11 +35,11 @@
   request context which contains the state of a running CMR."
   :execution-context)
 
-(defmethod get-providers :cli
-  [context]
-  (let [providers-url (format "%sproviders"
-                              (config/application-public-root-url :ingest))]
-    (util/endpoint-get providers-url {:accept mt/json})))
+;; (defmethod get-providers :cli
```

Comment:
We should delete the code that is no longer used.

Reply:
Do you need a space between %s and providers?